apps/docs/src/examples/getting-started/Scene.svelte

```diff
@@ -12,7 +12,7 @@
     MeshStandardMaterial,
     PerspectiveCamera
   } from 'three'
-  import { degToRad } from 'three/src/math/MathUtils'
+  import { degToRad } from 'three/src/math/MathUtils.js'
```

Comment:
For anyone else wanting to double check the [exports](https://github.com/mrdoob/three.js/blob/c2593ed3db121b17590068c638d5dc115e7496f9/src/math/MathUtils.js#L337)

yes, both `degToRad` and `DEG2RAD` are exports 🤷‍♂️

Reply:
Oh, this is not a suggestion, nevermind then :)

Reply:
regex is good 👍